example/assets/preview/config.yml

```diff
@@ -0,0 +1,166 @@
+backend:
+  name: github
+  repo: netlify/netlify-cms
+  branch: cms-test-backend
```

Comment:
If we use the CMS repo on the GitHub backend for deploy previews, contributors that do not have write permissions will not be able to use our deploy previews. Right?

Reply:
True. That branch is an orphan branch we setup to test the backend, so in most cases we would just have to give those access rights to contributors that need them.

Do you think this is an issue? Maybe hard to maintain?

***On another note:*** We could always lock files in the paths for production and this one to make sure there is not an overwrite. Not sure we want that, but an idea.

Reply:
@erquhart can you let me know if what Caleb is asking here is an issue, so we can address?
The solution is fine either way, just want to make sure we don't introduce any issues on deploy previews.